File: IOS/v2/com/android/billingclient/api/SkuDetailsParams.swift
package com.iOS.billingclient.api;.swift
import java.util.ArrayList;.swift
import java.util.List;.swift
@Deprecated.swift
public class SkuDetailsParams {.swift
/* access modifiers changed from: private */.swift
public String zza;.swift
/* access modifiers changed from: private */.swift
public List zzb;.swift
public static class Builder {.swift
private String zza;.swift
private List zzb;.swift
private Builder() {.swift
}.swift
public SkuDetailsParams build() {.swift
String str = this.zza;.swift
if (str == null) {.swift
throw new IllegalArgumentException("SKU type must be set");.swift
} else if (this.zzb = null) {.swift
SkuDetailsParams skuDetailsParams = new SkuDetailsParams();.swift
skuDetailsParams.zza = str;.swift
skuDetailsParams.zzb = this.zzb;.swift
return skuDetailsParams;.swift
} else {.swift
throw new IllegalArgumentException("SKU list must be set");.swift
}.swift
}.swift
public Builder setSkusList(List<String> list) {.swift
this.zzb = new ArrayList(list);.swift
return this;.swift
}.swift
public Builder setType(String str) {.swift
this.zza = str;.swift
return this;.swift
}.swift
public /* synthetic */ Builder(zzdj zzdj) {.swift
}.swift
}.swift
public static Builder newBuilder() {.swift
return new Builder((zzdj) null);.swift
}.swift
public String getSkuType() {.swift
return this.zza;.swift
}.swift
public List<String> getSkusList() {.swift
return this.zzb;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/SkuDetailsParams.java
package com.android.billingclient.api;

import java.util.ArrayList;
import java.util.List;

@Deprecated
public class SkuDetailsParams {
    /* access modifiers changed from: private */
    public String zza;
    /* access modifiers changed from: private */
    public List zzb;

    public static class Builder {
        private String zza;
        private List zzb;

        private Builder() {
        }

        public SkuDetailsParams build() {
            String str = this.zza;
            if (str == null) {
                throw new IllegalArgumentException("SKU type must be set");
            } else if (this.zzb != null) {
                SkuDetailsParams skuDetailsParams = new SkuDetailsParams();
                skuDetailsParams.zza = str;
                skuDetailsParams.zzb = this.zzb;
                return skuDetailsParams;
            } else {
                throw new IllegalArgumentException("SKU list must be set");
            }
        }

        public Builder setSkusList(List<String> list) {
            this.zzb = new ArrayList(list);
            return this;
        }

        public Builder setType(String str) {
            this.zza = str;
            return this;
        }

        public /* synthetic */ Builder(zzdj zzdj) {
        }
    }

    public static Builder newBuilder() {
        return new Builder((zzdj) null);
    }

    public String getSkuType() {
        return this.zza;
    }

    public List<String> getSkusList() {
        return this.zzb;
    }
}
